cernopendata_client/cli.py

```diff
@@ -127,7 +128,7 @@ def get_metadata(server, recid, doi, title, output_value):
 )
 @click.option(
     "--server",
-    default="http://opendata.cern.ch",
+    default=SERVER,
     type=click.STRING,
     help="Which CERN Open Data server to query? [default=http://opendata.cern.ch]",
```

Comment:
in the help as well, you can do stuff like:

```python
help="Which CERN Open Data server to query? [default={}]".format(SERVER),
```
```

Reply:
done 👍